src/components/Button/Button.tsx

```diff
@@ -1,8 +1,27 @@
 import React from 'react';
-import { Button as MuiButton, ButtonProps as MuiButtonProps } from '@material-ui/core';
+import { Button as MuiButton, ButtonProps as MuiButtonProps, makeStyles } from '@material-ui/core';
 
 export type ButtonProps = MuiButtonProps;
 
+const useStyles = makeStyles((theme) => ({
+  button: {
+    '&.MuiButton-containedPrimary': {
+      backgroundColor: theme.palette.primary.main,
+      padding: theme.spacing(1),
+    },
+    '&.MuiButton-contained': {
+      width: '158px',
```

Comment:
Width and height of the button cannot be hardcoded - we don't know how large the content of the button will be. Instead of specifying width and height, specify font styles and padding, so button size will be dynamic.